packages/twenty-front/src/modules/ui/utilities/hotkey/hooks/useGoToHotkeys.ts

```diff
@@ -5,13 +5,24 @@ import { AppHotkeyScope } from '../types/AppHotkeyScope';
 
 import { useSequenceHotkeys } from './useSequenceScopedHotkeys';
 
-export const useGoToHotkeys = (key: Keys, location: string) => {
+type GoToHotkeysProps = {
```

Comment:
changing the API to take an object which is better from a dev XP as parameters are named when using the hook

Reply:
Yes, I fully agree, Object is way more easier to debug and use. 

I always to that :)